scripts/ms

```diff
@@ -53,6 +58,19 @@ def parse(args):
 
 def init(opts):
     """Initialize some information based on the options & config"""
+
+    # setup dask
+    if opts.scheduler_file is not None:
+        from dask.distributed import Client
+        client = Client(scheduler_file=opts.scheduler_file)
+        print(client)
+    elif opts.nthreads > 1 or opts.n_procs > 0:
+        from dask.distributed import Client
+        client = Client(n_workers=opts.nprocs, n_threads=opts.nthreads)
+        print(client)
+    else:
+        client = None
+
```

Comment:
This is maybe a bit verbose but allows for a range of dask schedular configurations. The `schedular_file` option would allow for the user to use a previously launched dask cluster with many options. The nthreads/nprocs concept would launch a local cluster.